keylime/cloud_verifier_tornado.py

```
@@ -364,6 +364,25 @@ def verifier_db_delete_agent(session: Session, agent_id: str) -> None:
+    If the agent is already gone, just log and return.
+    """
+    try:
+        with session_context() as session:
+            agent = session.query(VerfierMain).filter_by(agent_id=agent_id).first()
+            if agent is not None and agent.operational_state == states.TERMINATED:  # pyright: ignore
+                logger.info("Agent %s was terminated during attestation, completing deletion", agent_id)
+                verifier_db_delete_agent(session, agent_id)
+            else:
+                logger.info("Agent %s was deleted during attestation, stopping poll cycle", agent_id)
+    except SQLAlchemyError as e:
+        logger.error("SQLAlchemy Error completing deletion for agent %s: %s", agent_id, e)
```

Ruff (0.15.12)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)